src/libsystemd-network/sd-dhcp6-lease.c

```diff
@@ -587,6 +588,83 @@ int sd_dhcp6_lease_get_captive_portal(sd_dhcp6_lease *lease, const char **ret) {
         return 0;
 }
 
+int dhcp6_lease_get_vendor_option(sd_dhcp6_lease *lease) {
+        int r;
+        sd_dhcp6_option **options = NULL;
+        size_t n_vendor_options;
+
+        assert(lease);
+
+        r = set_dump_sorted(lease->vendor_options, (void***)&options, &n_vendor_options);
+        if (r < 0)
+                return r;
+
+        return 0;
+}
+
+static int dhcp6_lease_insert_vendor_option(
+                sd_dhcp6_lease *lease,
+                uint16_t option_code,
+                const void *data,
+                size_t len,
+                uint16_t enterprise_id) {
+        _cleanup_(sd_dhcp6_option_unrefp) sd_dhcp6_option *option = NULL;
```

Comment:
Please insert an empty line between function arguments and variable declarations.

Reply:
Added empty line